server/src/test/java/org/opensearch/index/store/remote/filecache/FileCacheTests.java

```diff
@@ -280,11 +298,11 @@ private void putAndDecRef(FileCache cache, int path, long indexInputSize) {
         cache.decRef(key);
     }
 
-    private static class StubCachedIndexInput implements CachedIndexInput {
+    public static class StubCachedIndexInput implements CachedIndexInput {
```

Comment:
Out of curiosity, why it was promoted to public? (sealed test class makes sense)

Reply:
I used it within `TransportClearIndicesCacheActionTests.testInShardOperation` for creating a dummy file cache entry - seemed to be better alternative than creating another dummy class.